content/en/docs/tasks/debug-application-cluster/debug-service.md

```diff
@@ -213,6 +213,14 @@ In order to highlight the full range of configuration, the Service you created
 here uses a different port number than the Pods.  For many real-world
 Services, these values might be the same.
 
+## Any Network Policy Ingress rules affecting the target Pods?
+
+If you have deployed any Network Policy Ingress rules which may affect incoming
+traffic to `hostnames-*` Pods, these need to be reviewed.
+
+Please refer to [Network Policies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+for advanced details.
```

Comment:
```suggestion
Please refer to [Network Policies](/docs/concepts/services-networking/network-policies/) for more details.
```

Reply:
Hey @tengqm can you tell why we follow the url in a format `/docs/concept` and not `https://` as I have seen the Url format you suggested would give error in github while the other url won't give in github as well as actual website

Reply:
Oh my bad @tengqm, missed this. +1 adding.

Reply:
@AvineshTripathi 

if you use the absolute or full url (`https://kubernetes.io/docs/concept`), the link will always direct the browser to that path at *the latest main version*.

if you use the relative form (`/docs/concept`), the link will direct the browser to that path at the *version the user was previously browsing*. such as https://v1-20.docs.kubernetes.io/